### DISCUSSION TITLE
Problem with bridged networking?

### DISCUSSION THREAD

**Discussion by jalanh11640** [utmapp/UTM#5492] [Linux]
I created a ubuntu 22.0.4 server installation (the minimal installation, not the full) on a 2023 M2 Mac Mini running macOS Sonoma (public beta).  The UTM configuration defaulted to a network mode of 'Shared Network'. In this configuration, I was able to ssh to the instance from my Mac.

I decided to switch the Networking Mode to 'Bridged (Advanced)' and set the Bridged Interface to 'en1' (my wifi interface).  I kept the Emulated Network Card as 'virtio-net-pci.  When I booted the instance the IP matches the reservation I provided my router so it would appear that networking is configured.  From my Mac, I can ping the IP address without issue.  However, when I try to ssh to the instance ssh eventually times out.

Any suggestions on how to fix this?

Thanks,
Jeff

**Comment by pcjmfranken** [comment]
Try and see if enabling the "Isolate Guest from Host" option of "Bridged (Advanced)" does the trick for you. It's tucked away under the "Show Advanced Settings" panel by default:

<img width="816" alt="utm_network_bridged_isolated_host" src="https://github.com/utmapp/UTM/assets/17924090/449389bc-4c4b-4553-a9b2-3de50a0af2c8">

**Reply by jalanh11640** [reply]
Thanks for replying. I managed to get back to this today. I had uninstalled UTM. Today I reinstalled and bridged networking worked like a charm.

**Reply by pcjmfranken** [reply]
Interesting, guess we'll never know the cause. I hate forever mysteries 🥲

**Reply by derekhobden** [reply]
Ticking tat box worked for me

**Reply by dylanmc** [reply]
Also worked for me with Ubuntu 24.04. Thanks - that solution was counter-intuitive.

**Reply by pftbest** [reply]
This option helped me too, but it is very confusing. Is there any description what it actually does?
